pkg/operation/botanist/component/machinecontrollermanager/machine_controller_manager_test.go

```diff
@@ -325,6 +328,116 @@ var _ = Describe("MachineControllerManager", func() {
 				},
 			},
 		}
+
+		clusterRoleYAML := `apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: gardener.cloud:target:machine-controller-manager
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  - nodes/status
+  - endpoints
+  - replicationcontrollers
+  - pods
+  - persistentvolumes
+  - persistentvolumeclaims
+  verbs:
+  - create
+  - delete
+  - deletecollection
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - pods/eviction
+  verbs:
+  - create
+- apiGroups:
+  - extensions
```

Comment:
Do we still need `extensions` apiGroup, It's not served since v1.16, https://kubernetes.io/docs/reference/using-api/deprecation-guide/#deployment-v116